src/main/java/org/broadinstitute/consent/http/health/SendGridStatus.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.google.gson.Gson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SendGridStatus {
+    private Object page;
```

Comment:
We don't really need `page` for any logic, so it makes sense that this is just a plain object. Generally, I try to avoid using `Object` - I like clarity in the code and `Object` tells me literally nothing about what this field is really about. If you wanted to go the extra step, we could add an inner class in `SendGridStatus` that maps out the values in `page`. Consider this optional feedback.